tests/integration/targets/ome_application_certificate/tests/_download_multi_param_cert.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
the helper files extension will be .yml ?

Reply:
> the helper files extension will be .yml ?

I guess we can have any of these **.yaml** or **.yml**.